pkg/testutil/util.go

```diff
@@ -79,11 +79,18 @@ func IsYamlEqual(expectation *string, actualYamlByteArray *[]byte) bool {
 	}
```

Comment:
@mbaijal The use of the `os` commands for file and string manipulation is not OS cross-compatible and goes against best practices for Go. Please make it a high priority to replace any use of system commands with the appropriate Go packages ASAP.

Reply:
Absolutely, planning to do it in the next PR.